jobmon_gui/src/task_details_page/task_instance_table.tsx

```diff
@@ -19,10 +19,11 @@ export default function TaskInstanceTable({ taskInstanceData }) {
     const [showStdoutModal, setShowStdoutModal] = useState(false)
     const [showStderrModal, setShowStderrModal] = useState(false)
 
+    // ti_stderr_log is pulled from task_instance.stderr_log, ti_error_log_description is pulled from task_instance_error_log.description
```

Comment:
isn't the plan to deprecate the task_instance_error_log table entirely? What does this add if errors are logged to the task instance table already

Reply:
Logan said there's a chance we may keep error logs in both places. This PR comes from a help desk ticket Ezra created saying that he couldn't see his errors on the TaskDetails page (since he wasn't using 3.2.1).

Reply:
thanks for the context, but then rather than a separate column shouldn't the logic be something like "if the TaskInstance.stderr column is empty then pull from the TaskInstanceErrorLog table"? Rather than two columns representing the same thing with one being null

I imagine the server logic can default to the new schema with a fallback option to query for NULL error logs

Reply:
So this isn't creating two separate columns. There's a single pop up modal that is displaying the error messages. If the error message is NULL in one of the columns in the database it will simply not show at all in the GUI. Once all users move to >3.2.1 we may walk this change back and only display error logs from the Task Instance table.

Reply:
Based on [this](https://github.com/ihmeuw-scicomp/jobmon/pull/14/files#diff-a2f8582864a8bfa92b66b10b313805710a325cda9675c5db0ae9bcb193eaa3e6R216) implementation won't the pop up model display both the stderr_log and the error_log_description? 

Maybe it's just semantics since one will always be null but just asking if that's the correct pattern

Reply:
Semantics because one will always be null. The pattern could probably be better but it's most likely going to change once all versions are >3.2